<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataProcessClass</class>
 <widget class="QMainWindow" name="DataProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_root">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_res">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结果保存到：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ai">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>alignIter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_demo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>showDemo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_res">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择根目录：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_root">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_icp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ICP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_as">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>alignSimp</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dataprocess.qrc"/>
 </resources>
 <connections/>
</ui>
